File: output/Llama-2-100/avatar/Python/Java/codeforces_621_A.java
import java.util.Scanner;

public class codeforces_621_A {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        List<Integer> lst = new ArrayList<>();
        for (int i = 0; i < n; i++) {
            String str = sc.next();
            lst.add(Integer.parseInt(str.split(" ")[0]));
        }
        Collections.sort(lst);
        Collections.reverse(lst);
        int od = 0;
        for (int i = 0; i < n; i++) {
            if (lst.get(i) & 1 == 1) {
                od++;
            }
        }
        int sum = 0;
        boolean ok = false;
        for (int i = 0; i < n; i++) {
            if (lst.get(i) % 2 == 0) {
                sum += lst.get(i);
            } else {
                if (ok || od > 1) {
                    sum += lst.get(i);
                    od--;
                    ok = !ok;
                }
            }
        }
        System.out.println(sum);
    }
}

File: output/Llama-2-100/avatar/Java/Python/codeforces_621_A.py
def codeforces_621_A ( n ): List = [ ] for i in range ( n ): List.append ( int ( input ( ) ) ) odd = [ ] for i in List: if i % 2 == 0: sum += i else: odd.append ( i ) Collections.sort ( odd ) for i in odd: sum += i if len ( odd ) % 2 != 0: sum -= odd [ 0 ] print ( sum )

